src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super(color);
+        this.setRadius(radius);
+    }
+
+    @Override
+    public double area() {
+        return Math.PI * radius * radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getName() + ", area: " + area()
+                + " units, radius: " + radius + " units, color: " + getColor());
```

Comment:
```suggestion
        System.out.println("Figure: " + getName() + ", area: " + area()
                + " units, radius: " + radius + " units, color: " + color);
```
Don't use getters when you have direct access to class fields. fix in all similar places